stanford-earth/stanford_earth_helper
# EARTH-697: Earth Events

# READ FOR REVIEW

# Summary
- Adds Earth Events to Main Module

# Needed By (Date)
- Sooner than later. Config was merged in to SE3_BLT that relies on this.

# Urgency
- Sooner than later.

# Steps to Test

1. Check out `develop` or `fixup` of the SE3_BLT repo
2. Make sure that this branch is checked out in stanford_earth_helper
3. Run a blt sync
4. Ensure no errors

# Affected Projects or Products
- Config currently in SE3_BLT develop relies on this new module

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
